docs/install/iso-install.md

```diff
@@ -60,11 +60,19 @@ The following [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of an
 
 1. Configure network interface(s) for the management network. By default, Harvester creates a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or statically assigned.
 
+   ![config-network.png](/img/v1.2/install/config-network.png)
+
 	:::note
 	It is not possible to change the node IP throughout the lifecycle of a Harvester cluster. If using DHCP, you must ensure the DHCP server always offers the same IP for the same node. If the node IP is changed, the related node cannot join the cluster and might even break the cluster.
-	:::
 
-   ![config-network.png](/img/v1.2/install/config-network.png)
+	When using DHCP, the `option routers` of dhcp server is a **mandatory option**, which is used to install `default route` in the Harvester host. The node will fail to start without it.
+	For example:
```

Comment:
```suggestion
In addition, you are required to use the *routers* option (`option routers`) when configuring the DHCP server. This mandatory option, which specifies a list of IP addresses for routers in order of preference, is used to configure the default route on the Harvester host. Without the default route, the node will fail to start.  

	Example:
```

Reply:
btw: replace `create` with `add`, it is a sub-command key word in https://man7.org/linux/man-pages/man8/ip-route.8.html